security/src/main/resources/messages_en.properties

```diff
@@ -23,16 +23,19 @@ assign_permission_error_for_role=Unable to assign $ permission for $ role
 remove_permission_from_role_error=Unable to remove $ permission for the $ role
 remove_package_error=Unable to remove $ package from $ client
 role_remove_from_package_error=Unable to remove $ role from $ package
-client_password_policy_error=Password is violating the client's policy.
-capital_letters_missing=At least one capital letter is required.  
-small_letters_missing=At least one small letter is required.
-numbers_missing=At least one Number is required.
-special_characters_missing =At least one special character is required.
-spaces_missing=Spaces are not allowed.
-regex_mismatch=The password doesn't match the suggested pattern.
-min_length_error=Password should be at least $ characters long.
-max_length_error=Password should not be more than $ characters long.
-user_not_active=The user is not active.
-old_new_password_match=The password is matches the existing password.
-new_password_missing=A password is required to update the password.
-password_used_error=This password was used.
\ No newline at end of file
+client_password_policy_error=Password is violating the client's policy
+capital_letters_missing=At least one capital letter is required  
+small_letters_missing=At least one small letter is required
+numbers_missing=At least one Number is required
+special_characters_missing =At least one special character is required
+spaces_missing=Spaces are not allowed
+regex_mismatch=The password doesn't match the suggested pattern
+min_length_error=Password should be at least $ characters long
+max_length_error=Password should not be more than $ characters long
+user_not_active=The user is not active
+old_new_password_match=The password is matches the existing password
+new_password_missing=A password is required to update the password
+password_used_error=This password was used
+delete_permission_error=Delete this permission from roles to which it was associated
+delete_role_error=Delete this role from package to which it was assigned
+delete_package_error=Delete this package from clients to it was assigned
```

Comment:
The password matches the existing password
Provide a password to update
Cannot delete permissions associated with users and roles
Cannot delete roles related to packages and users
Cannot delete packages related to clients